libs/island-ui/core/src/lib/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -18,12 +17,13 @@ interface Props {
   label: string
   buttonText: string
   variant?: ColorVariant
-  onClickSubmit: () => void
   state?: State
   errorMessage?: string
+  onChange: (bla: any) => void
```

Comment:
maybe change 'bla' to something more descriptive?

Reply:
:'D

Reply:
Fixed in https://github.com/island-is/island.is/pull/1018/commits/743a7b5951c17dce9d8badd302088d8d209a7037